sway-lib-std/src/ops.sw

```diff
@@ -1743,36 +1744,124 @@ impl PartialEq for str {
 #[cfg(experimental_partial_eq = true)]
 impl Eq for str {}
 
-fn assert(v: bool) {
-    if !v {
-        __revert(0)
+impl u8 {
+    /// Returns whether a `u8` is set to zero.
+    ///
+    /// # Returns
+    ///
+    /// * [bool] -> True if the `u8` is zero, otherwise false.
+    ///
+    /// # Examples
+    ///
+    /// ```sway
+    /// fn foo() {
+    ///     let zero_u8 = u8::zero();
+    ///     assert(zero_u8.is_zero());
+    /// }
+    /// ```
+    pub fn is_zero(self) -> bool {
+        self == 0u8
     }
 }
 
-#[test]
-pub fn ok_str_eq() {
-    assert("" == "");
-    assert("a" == "a");
+impl u16 {
```

Comment:
If we are moving this, why to `ops.sw` and not to `primitives.sw`? I imagine it's the ordering of these modules.

Reply:
the implementation uses the eq operation